app/server/lib/ExternalStorage.ts

```diff
@@ -1,6 +1,8 @@
-import {ObjMetadata, ObjSnapshot, ObjSnapshotWithMetadata} from 'app/common/DocSnapshot';
 import log from 'app/server/lib/log';
 import {createTmpDir} from 'app/server/lib/uploads';
+import {isAffirmative} from 'app/common/gutil';
```

Comment:
Sorry for the repeated silly lint nag, but can you order 'app/common' before 'app/server' (alphabetically). Not all our source code does this, but a lot of it does.

Reply:
No problem!